gradle/libs.versions.toml

```diff
@@ -75,6 +75,9 @@ javax-inject = "1"
 
 # https://junit.org/junit4/
 junit4 = "4.13.2"
+# https://github.com/junit-team/junit-framework/releases
+junit-jupiter = "5.14.1"
+junit-platform = "1.14.1"
```

Comment:
Why not use jUnit6 directly?
Renovate will probably submit a PR for that once this one is merged.
And with jUnit 6, the modules use the same version, so only one version is necessary here.